satpy/tests/reader_tests/test_seviri_l1b_native.py

```diff
@@ -1248,7 +1265,144 @@ def test_read_header():
     dtypes = np.dtype([(k, t) for k, t in zip(keys, types)])
     hdr_data = np.array([values], dtype=dtypes)
 
-    with mock.patch("satpy.readers.seviri_l1b_native.np.fromfile") as fromfile:
+    with mock.patch("satpy.readers.seviri_l1b_native.fromfile") as fromfile:
         fromfile.return_value = hdr_data
         actual = recarray2dict(hdr_data)
     assert actual == expected
+
+
+@pytest.fixture(scope="session")
+def session_tmp_path(tmp_path_factory):
+    """Generates a single temp path to use for the entire session."""
```

Comment:
Function docstrings should use a "command"/imperative verb tense. I'm not sure why ruff in pre-commit isn't complaining about these:

https://docs.astral.sh/ruff/rules/non-imperative-mood/

All functions below this one seem to need the change too ("Generates" -> "Generate", "Creates" -> "Create", etc).

Reply:
Thanks for pointing out this. I just changed them to imperative mood. 

I think the reason that it is not picked up by `ruff` is that the convention is set to `google`:
https://github.com/pytroll/satpy/blob/d2146766c1580ba3459f4e0c59b4ae83c211eb81/pyproject.toml#L145-L146

If we want to have the imperative mood enabled by default we have to use either `numpy` or `pep257` instead.